resources/views/livewire/ldap-settings-form.blade.php

```diff
@@ -0,0 +1,715 @@
+<div>
```

Comment:
I removed the div from each input and placed it around all the inputs. Still seems to look the same. I will resolve this last, since it will take the most time

Reply:
thanks for the help on this @marcusmoore 👍